README.md

```diff
@@ -43,69 +41,14 @@ npm i ngx-sub-form
-
-Previously, `transformToFormGroup` _(which is now as you guessed it `toFormGroup`)_ was taking as the first parameter `obj: ControlInterface | null` and as a second one `defaultValues: Partial<FormInterface> | null`. This was pretty annoying as you needed to define a `getDefaultValues` method to provide your default values. Now you simply define your default values within the `formControls` function on each of the form controls as you'd expect. Behind the scenes, when the component is created for the first time we make a deep copy of those default values and apply them automatically if the root form or the sub form is being updated upstream with `null` or `undefined`.
-
-If you were previously using inheritance to set some defaults globally, for example on your root forms for the `handleEmissionRate` method, you cannot do that anymore and you'll need to define those on a per component basis! So if you were extending your own class, itself inheriting from a root form or a sub form, don't forget about that. We're considering passing a token through DI to be able to set some of those settings globally. But it's not done yet and give us feedback if you think it should.
-
-For root forms, the helper `DataInput` has been removed. It is now by default slightly more verbose to get the input data as you have to declare a `Subject` and push values into it yourself (by using either a setter on your input or the `ngOnChanges` hook). `DataInput` was originally created to reduce this boilerplate but as there are plenty of libraries available to transform an input into an observable, we let the choice to either do it manually or install a library on your side to transform the input into an observable for you.
-
-You can also have a look into our demo app located here: `src/app`. You'll find `main` and `main-rewrite` which are exactly the same applications but `main` is using the deprecated API (the one with inheritance) while `main-rewrite` is using the new one. As those 2 applications showcase all the features of ngx-sub-form you can easily find what you're looking for and compare both if we forgot to cover anything. Just as an FYI, we've kept both apps for now which are tested by the same E2E test suite to make sure that nothing got broken on the old API during the rewrite. When we decide to remove the old API we'll of course remove the demo implementation which is using the old API.
+| `@angular` version     | `ngx-sub-form` version                               |
+| ---------------------- | ---------------------------------------------------- |
+| <= `7`                 | <= `2.7.1`                                           |
+| `8.x`                  | `4.x`                                                |
+| `9.x`, `10.x`          | `5.1.2`                                              |
+| `11.x`, `12.x`, `13.x` | `6.0.0` (non breaking but new API available as well) |
+| `14.x`                 | `7.0.0`                                              |
```

Comment:
Angular 14 support actually landed on 6.0.0 https://github.com/cloudnc/ngx-sub-form/releases

The readme was incorrect before - the new api was available from 5.2.0

Reply:
Indeed, fixed